### PR TITLE
Update Go implementation to match latest Finalize change.

### PR DESCRIPTION
**Pull request by chris-wood**
cc @alxdavids @armfazh

### CODE REVIEW

File: go/oprf/oprf.go [hunk 1]
```diff
@@ -73,7 +74,7 @@ func (ev Evaluation) ToJSON(verifiable bool) ([]byte, error) {
 type Participant interface {
 	Ciphersuite() gg.Ciphersuite
 	Setup(string, gg.PrimeOrderGroup) (Participant, error)
-	Blind([]byte) (gg.GroupElement, *big.Int, error)
+	Blind([]byte) (gg.GroupElement, gg.GroupElement, *big.Int, error)
```
Comment (alxdavids) [inline go/oprf/oprf.go:77]: Do we want to be careful here? We're exposing an interface that is different to the interface exposed by the VOPRF draft. `BlindFixed` is used when generating test vectors, would there be a way of only modifying the interface for this function (which is not documented) and keeping the `Blind` interface the same?

Comment (chris-wood) [reply go/oprf/oprf.go:77]: My take was that this only changes the implementation, and not the actual interface. Though on hindsight, you're probably right. Let me rework this a bit. Stand by!

**Comment by chris-wood** [comment]
@alxdavids I reverted the `Blind` change -- please have a look!

File: go/oprf/oprf.go [hunk 2]
```diff
@@ -239,36 +240,43 @@ func (c Client) Setup(ciphersuite string, pogInit gg.PrimeOrderGroup) (Participa
 // Blind samples a new random blind value from ZZp and returns P=r*T where T is
 // the representation of the input bytes x in the group pog.
 func (c Client) Blind(x []byte) (gg.GroupElement, *big.Int, error) {
+	P, _, r, err := c.BlindInternal(x)
+	return P, r, err
+}
+
+// Blind samples a new random blind value from ZZp and returns P=r*T where T is
```
Comment (alxdavids) [inline go/oprf/oprf.go:247]: Rewrite comment to reference `BlindInternal`

Comment (chris-wood) [reply go/oprf/oprf.go]: Fixed!

**Pull request merged**